docs/Release_notes.md

```diff
@@ -1,6 +1,28 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.3.0, released TBD
+--------------------------------------------------------
+
+**Summary:**
+CARMAPlatform release version 3.3.0 is comprised of two key enhancements. Firstly the plugin manager is now able to return a list of services that match a user-requested capability. Second, there are new configurations for the Freightliner Cascadia, including lidar configurations which allow autoware to combine the two lidar devices' data into a single stream. 
```

Comment:
Update "Firstly" to "First" since you're saying "Second" on the next. Please always make sure to copy and paste the Release Notes on Word Document for any other spelling/grammar errors that it may catch. The textpad is just not good at doing automated grammar check.